src/linter.rs

```diff
@@ -157,6 +165,11 @@ impl LinterBuilder {
     self.rules = rules;
     self
   }
+
+  pub fn plugins(mut self, plugins: Option<Box<dyn Plugins>>) -> Self {
```

Comment:
Maybe take a vector of plugins instead? `Vec<Box<dyn Plugin>>`

Reply:
done :)